### DISCUSSION TITLE
OpenHantec6022 weird behaviour

### DISCUSSION THREAD

**Discussion by asaidac** [OpenHantek/OpenHantek6022#345] [Q&A]
Hello
I need some assistance.
I've compiled .deb package in ArchLinux.
All good - but...
The device window (main scope window) extends for about 3 screen in with and all my attempts to resize it are failing.
2nd problem (maybe related to previous one) - I can not calibrate the probes ( I am following the procedure by shorting them) - both probes.

If someone will tell me to use Debian for compilation I will (still waiting for version 12)
Thank you.
Adrian

**Comment by Ho-Ro** [comment]
Please describe your system setup - window manager, X11? wayland? strange screen style?
A picture / screenshot would also help to narrow down your issue.

**Comment by asaidac** [comment]
ooo:                     ------------
                  yoooo/                    OS: ArcoLinux
                 yooooooo                   Kernel: 6.1.11-arch1-1
                yooooooooo                  Uptime: 1 day, 15 hours, 1 min
               yooooooooooo                 Packages: 1566 (pacman)
             .yooooooooooooo                Shell: bash 5.1.16
            .oooooooooooooooo               Resolution: 1600x900
           .oooooooarcoooooooo              DE: Xfce 4.18
          .ooooooooo-oooooooooo             WM: Xfwm4
         .ooooooooo-  oooooooooo            WM Theme: Default
        :ooooooooo.    :ooooooooo           Theme: Adwaita-dark [GTK2],
Arc-Dark [GTK3]
       :ooooooooo.      :ooooooooo          Icons: Sardi-Arc [GTK2/3]
      :oooarcooo         .oooarcooo         Terminal: xfce4-terminal
     :ooooooooy           .ooooooooo        Terminal Font: Terminus 12
    :ooooooooo   /ooooooooooooooooooo       CPU: Intel i5-2520M (4) @
3.200GHz
   :ooooooooo      .-ooooooooooooooooo.     GPU: AMD ATI Radeon HD
6970M/6990M
  ooooooooo-             -ooooooooooooo.    Memory: 3009MiB / 7902MiB (38%)
 ooooooooo-                 .-oooooooooo.
ooooooooo.                     -ooooooooo

**Comment by asaidac** [comment]
Here is a better picture - I can not resize the window horizontally.

Thanks

**Reply by Ho-Ro** [reply]
>  I can not resize the window horizontally.

Which part of the scope *eats up* the screen space?
Has the text a *normal* size or is it huge? In this case you could try to use a small font size with option e.g. `--size 6` or `--size 8`.

**Comment by asaidac** [comment]
Right side

**Comment by asaidac** [comment]
Hello
Sort of good news.
Using the following WM I am able to control the OpenHantek window size.

                 ooo:                     ------------
                  yoooo/                    OS: ArcoLinux
                 yooooooo                   Kernel: 6.1.11-arch1-1
                yooooooooo                  Uptime: 13 mins
               yooooooooooo                 Packages: 1566 (pacman)
             .yooooooooooooo                Shell: bash 5.1.16
            .oooooooooooooooo               Resolution: 1600x900
           .oooooooarcoooooooo              WM: awesome
          .ooooooooo-oooooooooo             Theme: Arc-Dark [GTK2/3]
         .ooooooooo-  oooooooooo            Icons: Sardi-Arc [GTK2/3]
        :ooooooooo.    :ooooooooo           Terminal: xfce4-terminal
       :ooooooooo.      :ooooooooo          Terminal Font: Terminus 12
      :oooarcooo         .oooarcooo         CPU: Intel i5-2520M (4) @
3.200GHz
     :ooooooooy           .ooooooooo        GPU: AMD ATI Radeon HD
6970M/6990M
    :ooooooooo   /ooooooooooooooooooo       Memory: 768MiB / 7902MiB (9%)
   :ooooooooo      .-ooooooooooooooooo.
  ooooooooo-             -ooooooooooooo.
 ooooooooo-                 .-oooooooooo.
ooooooooo.                     -ooooooooo

Not sure how to control the app window size from other (previously
documented) WM
If you think I should close the QA let me know.
Thank you for checking
